user-manual/metadata/METS.rst

```diff
@@ -0,0 +1,182 @@
+.. _METS_schema:
+
+======================
+METS in Archivematica
+======================
+
+General description
+--------------------
+
+The Metadata Encoding & Tranmission Standard, or `METS`_, is a schema for
+encoding various metadata, expressed in XML . It essentially acts a wrapper
+around other metadata standards, such as PREMIS and Dublin Core that are used in
+Archivematica.
+
+The METS file in Archivematica will have a basic generic structure that is
+comprised of the following sections:
+
+* metsHdr (METS header) (1 only)
+* dmdSec (descriptive metadata section) (1 or more)
+* fileSec (file section) (1 only)
+* structMap (structural map) (1 or more)
+
+.. literalinclude:: scripts/metsfile.xml
+   :language: bash
+   :lines: 1-26
+
+Below is a more detailed outline of the above sections present in AIPs created
+in Archivematica.
+
+Skip to:
+
+* :ref:`<metsHdr> <metshdr>`
+* :ref:`<dmdSec> <dmdsec>`
+* :ref:`<amdSec> <amdsec>`
+* :ref:`<fileSec> <filesec>`
+* :ref:`<structMap> <structmap>`
+
+
+.. _metshdr:
+
+<metsHdr>
+^^^^^^^^^
+
+There will be one METS header section for each METS file. It will contain a
+CREATEDATE attribute.
+
+.. code:: xml
+
+  <metsHdr CREATEDATE=”2019-06-18T23:52:11”/>
+
+If the AIP has been reingested, the metsHder section will also contain a
+LASTMODDATE.
+
+.. code:: xml
+
+  <metsHdr CREATEDATE=”2019-06-18T23:52:11” LASTMODDATE=”2019-06-27T00:28:56”/>
+
+
+.. _dmdsec:
+
+<dmdSec>
+^^^^^^^^
+
+There will be one dmdSec for the whole AIP, which contains PREMIS Intellectual
+Entity information.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 15-35
+
+There may also be one or more dmdSecs for descriptive metadata. The dmdSecs are
+numbered dmdSec_1, dmdSec_2, etc. and contain Dublin Core metadata as a default.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 64-91
+
+MDTYPE will be **DC** for Dublin Core metadata; if another metadata standard is
+included then the MDTYPE will be **OTHER**. 
+
+If the user does not enter any DC metadata during transfer/ingest and no DC
+metadata was included with the transfer (e.g. as a metadata.csv file, see
+:ref:`Import metadata <import-metadata>`), there will be no additional dmdSecs.
+
+
+.. _amdsec:
+
+<amdSec>
+^^^^^^^^^
+
+There is one amdSec for each object and wraps PREMIS entities as follows:
```

Comment:
I can't remember without looking, will there be one for each original object and preservation object? Is it worth adding that clarification and/or detail do you think?

Reply:
yes, I will expand on that for clarity.